CHANGES.txt

```diff
@@ -5,6 +5,9 @@ Changes for Crate
 Unreleased
 ==========
 
+ - Selecting ``os['timestamp']`` multiple times from ``sys.nodes`` within a
```

Comment:
this fix actually make os['timestamp'] return the actual time of each nodes clock, before it was the clock of the request handler ...  pls ensure that this is clear in the changelog